test/dynamo/test_misc.py

```diff
@@ -6750,6 +6750,82 @@ def main_generator(t_list):
         self.assertEqual(counter.frame_count, 0)
 
 
+    def test_itertools_accumulate_tensors_default_sum(self):
+        def fn(a, b, c, d, x):
+            l = [a, b, c, d, x]
+            for i, t in enumerate(l):
+                l[i] = t * x
+            return itertools.accumulate(l)
+
+        t_list = [torch.tensor([i]) for i in range(4)]
+        x = torch.randn([2, 2])
+        eager = fn(*t_list, x)
+
+        counter = CompileCounter()
+        compiled_fn = torch._dynamo.optimize(counter)(fn)
+        compiled = compiled_fn(*t_list, x)
+
+        self.assertEqual(list(eager), list(compiled))
+        self.assertEqual(counter.frame_count, 1)
+
+    def test_itertools_accumulate_tensors_builtins(self):
+        for builtin_op in [operator.mul, operator.sub, operator.pow]:
+            torch._dynamo.reset()
```

Comment:
why do you need a reset here? shouldn't we properly cache miss on new builtin op?

Reply:
Yup! Don't need it!

Reply:
oh, ok, great.

well there is one more reset to remove but ill stamp.